src/styles/vocabulary_icons.scss

```diff
@@ -61,19 +61,19 @@ $icons: (
   "plus": "\ea0d",
   "minus": "\ea0e",
 
-  "cc-logo": "\ec00",
```

Comment:
Please also arrange them in order of code point, separating the standard from the non-standard with a blank line.